libs/designsystem/accordion/src/accordion-item.component.spec.ts

```diff
@@ -50,6 +50,14 @@ describe('AccordionItemComponent', () => {
         visibility: 'visible',
       });
     });
+    it('should emit the toggle event when expanded', () => {
```

Comment:
Consider code readability:
```suggestion

    it('should emit the toggle event when expanded', () => {
```